about.html

```diff
@@ -12,57 +12,80 @@
 </head>
 
 <body>
-    <div class="container about-page">
+   <div class="container2">
```

Comment:
If the container on this about.html page and the container on the index.html page are going to be treated the same, you can keep the same class name. :)